content/en/docs/concepts/services-networking/service.md

```diff
@@ -642,9 +642,9 @@ You must enable the `ServiceLBNodePortControl` feature gate to use this field.
 
 #### Specifying class of load balancer implementation {#load-balancer-class}
 
-{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.22" state="beta" >}}
 
-Starting in v1.21, you can optionally specify the class of a load balancer implementation for
+From Kubernetes v1.21 onwards, you can optionally specify the class of a load balancer implementation for
 `LoadBalancer` type of Service by setting the field `spec.loadBalancerClass`.
 By default, `spec.loadBalancerClass` is `nil` and a `LoadBalancer` type of Service uses
 the cloud provider's default load balancer implementation. 
```

Comment:
We probably want to specify this is only true when cloud provider integration is enabled:
```
By default, `spec.loadBalancerClass` is `nil` and a `LoadBalancer` type of Service uses
the cloud provider's default load balancer implementation if the cluster is configured with
a cloud provider using the `--cloud-provider` component flag.
```

or similar